pages/import.vue

```diff
@@ -67,37 +67,44 @@
       </v-card>
 
       <!-- Results -->
-      <v-card v-for="(result, i) in results" :key="i" class="pa-2 my-6">
+      <v-card
+        v-for="(result, resultIndex) in results"
+        :key="resultIndex"
+        class="pa-2 my-6"
+      >
         <v-card-title class="justify-center">{{ result.title }}</v-card-title>
         <v-row>
           <v-col
-            v-for="(specFile, j) in allVegaFiles[i]"
-            :key="j"
+            v-for="(specFile, vegaIndex) in allVegaFiles[resultIndex]"
+            :key="vegaIndex"
             style="text-align: center"
           >
             <unit-vega-viz
               :spec-file="specFile"
-              :values="allProcessedItems[i][j]"
-              :div-id="`viz-${i}-${specFile.name}`"
+              :data="processResultForVega(result.result, specFile)"
+              :div-id="`viz-${resultIndex}-${specFile.name}`"
             />
           </v-col>
         </v-row>
         <v-row
-          v-for="(graphName, index) in allVueGraphNames[i]"
-          :key="`viz-${i}-${index}`"
+          v-for="(graphName, vizVueIndex) in allVueGraphNames[resultIndex]"
+          :key="`viz-${resultIndex}-${vizVueIndex}`"
         >
           <v-col>
-            <vue-graph-by-name :graph-name="graphName" :values="allItems[i]" />
+            <vue-graph-by-name :graph-name="graphName" :data="result.result" />
```

Comment:
Let's be careful with this kind of change because it breaks the import for zips that were generated before this PR. Fortunately we haven't used the import feature in production yet.